docs/tutorials/local_iteration_series/local-Iteration-testing-multiplayer-games-locally.md

```diff
@@ -0,0 +1,62 @@
+---
+id: local_iteration_testing_locally
+title: Local Iteration - Testing multiplayer games locally
+description: Guide covering the available workflows for testing multiplayer games locally.
+---
+
+Testing a multiplayer game presents unique challenges to developers. 
```

Comment:
What are the challenges?

Reply:
I DMed you a link to a conversation on some of these issues. You might want to sync with @becksebenius-unity to have a more complete list

Reply:
things like being able to debug, inspect, change client/server scenes. Listing these out could be interesting to users

Reply:
@SamuelBellomo lmk if you like the new phrasing better